packages/bpk-component-boilerplate/src/BpkBoilerplate.tsx

```diff
@@ -23,17 +23,17 @@ import STYLES from './BpkBoilerplate.module.scss';
 const getClassName = cssModules(STYLES);
 
 export type Props = {
-  className?: string | null;
   [rest: string]: any; // Inexact rest. See decisions/inexact-rest.md
 };
-const BpkBoilerplate = ({ className = null, ...rest }: Props) => {
-  const classNames = getClassName('bpk-boilerplate', className);
+const BpkBoilerplate = ({ ...rest }: Props) => {
+
+  // eslint-disable-next-line no-param-reassign
```

Comment:
I think you could avoid it by destructuring `className` separately from `rest`, but I guess `delete` is more explicit.

Reply:
Yeah we did used to have that but then its encouraging use or showing it could be available whereas here we want to prohibit it completely

Reply:
As in 

```
{ className, ...rest }: Props
```

Reply:
We want to avoid showing it could be in the props and used, so doing it explicitly with force shows if someone tries to build with it later on it would through an IDE error

Reply:
If we drop `className` it can still be passed via `rest` so it could still be included, and this forces it out completely so that it can't be spread via `...rest`